include/bluetooth/conn.h

```diff
@@ -299,6 +299,9 @@ struct bt_conn_le_info {
 #endif /* defined(CONFIG_BT_USER_DATA_LEN_UPDATE) */
 };
 
+/* Multiply bt 1.25 to get MS */
+#define BT_CONN_INTERVAL_TO_MS(interval) ((interval) * 5 / 4)
```

Comment:
This may seem a bit cumbersome, but I'd really prefer if the change to this macro (in this file and the samples) were done in a separate commit than the L2CAP commits.